.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
- [ ] Please the user can't add data by himself to the app make sure that you are using the right API URL, I see that you are creating a new game each time the user submits data instead of adding the data to the API for the game link as the doc in the project specified.![image](https://user-images.githubusercontent.com/74744787/224451202-a1faef42-badb-4d60-8f14-5f2deb9fb8bb.png)